packages/phpstan-rules/packages/symfony/src/Rules/RequireNamedCommandRule.php

```diff
@@ -123,4 +135,14 @@ private function isInNonAbstractCommand(Scope $scope): bool
 
         return $classReflection->isSubclassOf(Command::class);
     }
+
+    private function hasAsCommandAttribute(ClassMethod|Node $node): bool
```

Comment:
it always `ClassMethod` based on `getNodeTypes()` so `ClassMethod` is enough

Reply:
yes, sorry, I didn't see your first change suggestion. It should be okay, now. Thanks !